migrations/1578329693694-disable-team-created-by-students-all-schools.js

```diff
@@ -0,0 +1,47 @@
+const mongoose = require('mongoose');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module 'mongoose'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4774872)